src/main/java/com/datapipe/jenkins/vault/configuration/VaultConfiguration.java

```diff
@@ -130,6 +151,9 @@ public void setEngineVersion(Integer engineVersion) {
 
         public static final int DEFAULT_ENGINE_VERSION = 2;
 
+        public static final File DEFAULT_TRUSTSTORE =
+            new File(System.getProperty("java.home"), "lib/security/cacerts");
```

Comment:
This is already the default? so you can remove it.

Reply:
It's the default truststore location for the JRE, it doesn't look like javax.net.ssl.trustStore is set by default.

Reply:
This sslConfig goes into the default javax.net.ssl which will ensure cacerts file is loaded.

Reply:
https://github.com/BetterCloud/vault-java-driver#ssl-config <- BetterCloud says you have to provide your own for this one. I tried using javax.net.ssl.trustStore to get the trust store path to provide and it was returning null on my jenkins instance and my ubuntu box, so I'm not sure that's reliable without passing -Djavax.net.ssl.trustStore=... on the command line.

Reply:
Could you please add a test? Should be able to use testcontainer to setup vault with self signed certificate to confirm the use case. Perhaps it would be best to upstream to use the default trust store if that is not the case.

Reply:
Could you please add a test? Should be able to use testcontainer to setup vault with self signed certificate to confirm the use case. Perhaps it would be best to upstream to use the default trust store if that is not the case.

Reply:
Sure. I might as well add a check for existence before passing it to the sslcontext as well.